src/services/dataService.js

```diff
@@ -0,0 +1,80 @@
+import * as axios from 'axios';
+import {
+  API_AUTH_URL,
+  API_BASE_URL,
+  DATA_TYPES,
+  API_SEARCH_LOCATION_DATA_TYPE,
+  API_GET_CURRENT_WEATHER_DATA_TYPE,
+  API_GET_FORECAST_DATA_TYPE
+} from '../constants/constants';
+
+const instance = axios.create({
+  withCredentials: true,
+  baseURL: API_BASE_URL
+});
+
+export const weatherAPI = {
+  token: null,
+  getAuthHeaders() {
```

Comment:
I would rather name this function getHeaders. There may be also other headers later. Plus, why should it return them wrapped in another object? (see comment below)